lib/src/model/search_result.dart

```diff
@@ -15,6 +15,9 @@ class SearchResult extends JsonObject {
   @JsonKey(name: 'count', fromJson: JsonObject.parseInt)
   final int? count;
 
```

Comment:
// page_count is the number of products requested per page. Please be mindful of server resources, and keep that below 200.

Reply:
> page_count is the number of products requested per page.

AFAIK it's rather the number of products _found_ within that result page.